package.json

```diff
@@ -70,7 +70,6 @@
     "redux": "^4.1.0",
     "redux-devtools-extension": "^2.13.8",
     "redux-firestore": "sparkletown/redux-firestore#fix/introduce-batch-store-updates-on-addition",
-    "redux-thunk": "^2.3.0",
```

Comment:
[note] No longer needed as a direct dependency because `@reduxjs/toolkit` handles it for us.

Reply:
Nice 💯